src/Engine/Systems/RenderSystem.cpp

```
@@ -64,13 +65,38 @@ void RenderSystem::init(const Geometry::GeometryData *geomData, const Ctx::Offsc
 
 	m_sessionResourceID = m_cleanupManager->createCleanupGroup();
 
+	initOrbitVertexArray();
 	initGlobalBuffers();
 	createVisualizers();
 	initVisualizerCmdBufSets();
 	initVisualizers();
```

Cppcheck (2.20.0)

[performance] Function parameter 'queueFamilies' should be passed by const reference.

(passedByValue)